1.10/tutorials/dcos-101/app2.md

```diff
@@ -11,52 +11,41 @@ menu_order: 5
-       * Because redis is running in a Docker container, you need to list all Docker containers using the `docker ps` command to get the *ContainerID*.
-         * Connect to a bash session to the running container: `sudo docker exec -i -t CONTAINER_ID  /bin/bash`
-         * Start the redis CLI: `redis-cli`
-         * Check value is there: `get <newkey>`
+      
+      This should return you the raw HTML response from app2's webserver.
+  
+
+Accessing the app from within the cluster and viewing the raw HTML response proves our application is running, but in the real world you want to expose the app to the public. In the next part of this tutorial you will do exactly that. 
 
 # Outcome
- You have deployed a second app that uses the native Mesos containerizer. You used Marathon-LB to expose the app to the public and added a new key to redis using the web frontend.
+ You have deployed a second app that uses the native Mesos containerizer.
 
 # Deep Dive
-You have now deployed apps in two different ways: using Docker (app1) and natively (app2).
-Let us explore the differences in some more detail.
-DC/OS uses [containerizers](/docs/1.10/deploying-services/containerizers/) to run tasks in containers. Running tasks in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically. DC/OS supports the Mesos containerizer types DC/OS Universal container runtime and Docker containerizer.
+You have now deployed apps in two different ways: 
+
+1. Using Docker (app1)
+1. Using the native Universal Container Runtime (app2)
+
+Let's explore the differences in some more detail.
+
+DC/OS uses [containerizers](/docs/1.10/deploying-services/containerizers/) to run tasks in containers. Running tasks in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically. DC/OS supports three types of containerizers - the Mesos containerizer, the DC/OS Universal Container Runtime, and the Docker containerizer.
 
-For your first app, you actually used a Docker container image to package app1's depencies (remember: never rely on depencies being installed on an agent!) and then used the Docker containerizer to execute it. As the Docker containerizer internally uses the [Docker runtime](https://docs.docker.com/engine/userguide/intro/), you effectively used the Docker runtime.
+For your first app, you used a Docker container image to package app1's dependencies ( Remember: never rely on dependencies being installed on an agent! ) and then used the Docker containerizer to execute it. As the Docker containerizer internally uses the [Docker runtime](https://docs.docker.com/engine/userguide/intro/), you also used the Docker runtime.
```

Comment:
Suggest: "Because the Docker containerizer uses the Docker runtime internally,..."